server/src/test/java/io/crate/expression/scalar/timestamp/TimezoneFunctionTest.java

```diff
@@ -127,4 +127,9 @@ public void testEvaluateValidTimestamp() {
         );
         assertEvaluate("timezone('Europe/Madrid', x)", 257484600000L, Literal.of(257488200000L));
     }
+
+    @Test
+    public void test_timezone_doesnt_truncate_millis_behaves_like_now_in_utc() {
+        assertEvaluate("NOW() = timezone('UTC', current_timestamp(3))", true);
```

Comment:
Minor nit: Maybe we should compare the outcome against a literal instead to really test the correct behaviour?
Otherwise it could this be wrong if `NOW` is wrong.

Reply:
> if NOW is wrong.
good point, thanks. updated test